x-pack/plugins/security_solution/public/management/services/exceptions_list/exceptions_list_api_client.ts

```diff
@@ -92,7 +96,10 @@ export class ExceptionsListApiClient {
     listId: string,
     listDefinition: CreateExceptionListSchema
   ): ExceptionsListApiClient {
-    if (!ExceptionsListApiClient.instance.has(listId)) {
+    if (
+      !ExceptionsListApiClient.instance.has(listId) ||
+      !ExceptionsListApiClient.instance.get(listId)?.isHttp(http)
```

Comment:
I don't think you need the `?` here now that you added the method.

Reply:
`ExceptionsListApiClient.instance.get(listId)` can be undefined at this point